src/components/NotificationsModal/notifications/SomeoneAdmiredYourFeedEvent.tsx

```diff
@@ -0,0 +1,96 @@
+import { SomeoneAdmiredYourFeedEventFragment$key } from '__generated__/SomeoneAdmiredYourFeedEventFragment.graphql';
+import { useFragment } from 'react-relay';
+import { graphql } from 'relay-runtime';
+
+import { BaseM } from '~/components/core/Text/Text';
+import HoverCardOnUsername from '~/components/HoverCard/HoverCardOnUsername';
+import { CollectionLink } from '~/components/NotificationsModal/CollectionLink';
+import { SomeoneAdmiredYourFeedEventQueryFragment$key } from '~/generated/SomeoneAdmiredYourFeedEventQueryFragment.graphql';
+
+type SomeoneAdmiredYourFeedEventProps = {
+  notificationRef: SomeoneAdmiredYourFeedEventFragment$key;
+  queryRef: SomeoneAdmiredYourFeedEventQueryFragment$key;
+};
+
+export function SomeoneAdmiredYourFeedEvent({
+  notificationRef,
+  queryRef,
+}: SomeoneAdmiredYourFeedEventProps) {
+  const query = useFragment(
+    graphql`
+      fragment SomeoneAdmiredYourFeedEventQueryFragment on Query {
+        ...HoverCardOnUsernameFollowFragment
+      }
+    `,
+    queryRef
+  );
+
+  const notification = useFragment(
+    graphql`
+      fragment SomeoneAdmiredYourFeedEventFragment on SomeoneAdmiredYourFeedEventNotification {
+        count
+
+        feedEvent {
+          eventData {
+            ... on CollectionCreatedFeedEventData {
+              collection {
+                ...CollectionLinkFragment
+              }
+            }
+
+            ... on CollectorsNoteAddedToCollectionFeedEventData {
+              collection {
+                ...CollectionLinkFragment
+              }
+            }
+
+            ... on TokensAddedToCollectionFeedEventData {
+              collection {
+                ...CollectionLinkFragment
+              }
+            }
+          }
+        }
+
+        admirers(last: 1) {
+          edges {
+            node {
+              ...HoverCardOnUsernameFragment
+            }
+          }
+        }
+      }
+    `,
+    notificationRef
+  );
+
+  const count = notification.count ?? 1;
+  const firstAdmirer = notification.admirers?.edges?.[0]?.node;
+
+  return (
+    <>
+      <BaseM>
+        <strong>
+          {count > 1 ? (
+            <>{notification.count} collectors</>
+          ) : (
+            <>
+              {firstAdmirer ? (
+                <HoverCardOnUsername userRef={firstAdmirer} queryRef={query} />
+              ) : (
+                'Someone'
```

Comment:
this is interesting- in order to admire something, you need to be logged in. so being anon / "someone" should not be reachable

not a big deal though, don't think we need to report error

Reply:
Just a sane fallback incase graphql is failing